src/views/wedo2/wedo2.jsx

```diff
@@ -44,6 +44,14 @@ var Wedo2 = React.createClass({
                                 </h4>
                                 <p>
                                     <FormattedHTMLMessage id='wedo2.installText' />
+                                    <br />
+                                    <a href="https://itunes.apple.com/WebObjects/MZStore.woa/wa/viewSoftware?id=1084869222&mt=12">
```

Comment:
👍 was wondering if we wanted to leave it untouched until using the config, but sounds good to me!